src/button/src/components/styled/NeutralButtonStyles.js

```diff
@@ -0,0 +1,44 @@
+import { css } from 'styled-components';
+
+import { NEUTRALS, WHITE } from '../../../../colors';
+import { getStyleVariation } from '../../../../utils/StyleUtils';
+
+// border-top + border-bottom + padding-top + padding-bottom + line-height = 40px
+const fontColorVariation = getStyleVariation('fontColor', {
```

Comment:
I see. So instead, provide it as an inline style prop and do away with the `fontColor` prop? I guess that makes sense, since using the prop is essentially doing the same thing.
```
<Button
        mode="neutral"
        onClick={action('button clicked')}
        style={{ color: WHITE }}>
     Neutral
</Button>

Reply:
```
<Button
        mode="neutral"
        onClick={action('button clicked')}
        fontColor="rebeccapurple">
     Neutral
</Button>
```

Reply:
Oh! Yaa I like that better. Using "light" as prop option felt kinda silly, since the color is just white.